package.json

```diff
@@ -72,7 +73,7 @@
     "tslint-microsoft-contrib": "5.0.3",
     "typescript": "2.8.1",
     "url-loader": "1.0.1",
-    "webpack": "4.4.1",
+    "webpack": "^4.10.2",
```

Comment:
Can we pin the versions? :D

Reply:
well the lock file kinda does this already ... but ok, one sec :)

Reply:
done ... I intentionally left the `npm` only pinned to ^6.